From 0f1b5748e904019023badb4c7bbbcb9a48b0f635 Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Sat, 19 Jul 2014 04:30:23 -0400
Subject: [pci] msi: Fix pci_msix_vec_count() htmldocs failure

Message-id: <20140719043023.19438.50392.stgit@gir.stowe>
Patchwork-id: 86035
O-Subject: [RHEL7.1 PATCH 73/81] PCI/MSI: Fix pci_msix_vec_count() htmldocs failure
Bugzilla: 1110895
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1110895
Upstream Status: 75ce2d53cef8ea964634583ba45d6323ba5be243

commit 75ce2d53cef8ea964634583ba45d6323ba5be243
Author: Masanari Iida <standby24x7@gmail.com>
Date:   Thu Feb 13 10:47:44 2014 -0700

    PCI/MSI: Fix pci_msix_vec_count() htmldocs failure

    An empty line in msi.c caused "make htmldocs" failure:

      Warning(/home/iida/Repo/linux-next//drivers/pci/msi.c:962): bad line:

    Fixes: ff1aa430a2fa ("PCI/MSI: Add pci_msix_vec_count()")
    Signed-off-by: Masanari Iida <standby24x7@gmail.com>
    Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/pci/msi.c b/drivers/pci/msi.c
index cfce591..7d2592d 100644
--- a/drivers/pci/msi.c
+++ b/drivers/pci/msi.c
@@ -988,7 +988,6 @@ EXPORT_SYMBOL(pci_disable_msi);
 /**
  * pci_msix_vec_count - return the number of device's MSI-X table entries
  * @dev: pointer to the pci_dev data structure of MSI-X device function
-
  * This function returns the number of device's MSI-X table entries and
  * therefore the number of MSI-X vectors device is capable of sending.
  * It returns a negative errno if the device is not capable of sending MSI-X
-- 
1.7.1